charts/posthog/values.yaml

```diff
@@ -48,6 +48,13 @@ events:
     # -- Max pods for the events stack HorizontalPodAutoscaler.
     maxpods: 10
 
+  securityContext:
```

Comment:
Nit: any chance we could split this into `containerSecurityContext` and `podSecurityContext `?

Reply:
Done